lib/timers.js

```diff
@@ -82,11 +82,11 @@ function listOnTimeout() {
 
   debug('timeout callback ' + msecs);
 
-  var now = Timer.now();
-  debug('now: ' + now);
-
   var first;
   while (first = L.peek(list)) {
+    var now = Timer.now();
```

Comment:
Good point, I added a `_addedTimer` property on timers lists to determine when a timer has been added to the list. There are other ways to do that, but this one seemed to be the more lightweight and less intrusive.